proto/lighthouse-result.proto

```diff
@@ -55,6 +55,8 @@ enum LighthouseError {
   DNS_FAILURE = 19;
   // A timeout in the initial connection to the debugger protocol.
   CRI_TIMEOUT = 20;
+  // Layout Shift trace events are found but without data
+  NO_LAYOUT_SHIFT = 21;
```

Comment:
since this is not a runtimeError I believe we don't need to add it to this list

Reply:
@exterkamp can you confirm

Reply:
Unless it is in `lh-error.js` we don't need it here. @brendankenny should be right.

Reply:
it is there

Reply:
We should update the comment in lh-error cause we've had this conversation like three or four times :) We just did this for `NO_LCP`, for instance.

-proto-wise: The only place the proto `LighthouseError` shows up in the proto is as a runtime error, so only runtime errors need to be allowable values for it.

- audit-wise: We don't expose LHErrors (or any errors) thrown in individual audits--only inserting their message into the audit error explanation--unless the error is marked as `runtimeError: true`. There's no other way for it to escape into the lhr, and so no need to declare it in the proto.

Reply:
> We don't expose LHErrors (or any errors) thrown in individual audits. unless the error is marked as `runtimeError: true`

Actually, jk, not even then. lhr runtimeErrors only [come from](https://github.com/GoogleChrome/lighthouse/blob/8b4d36c318b852ac12d7fb5b7f4c8bee9a636dd2/lighthouse-core/runner.js#L359-L383) artifacts and pageloadErrors, [never audits](https://github.com/GoogleChrome/lighthouse/blob/8b4d36c318b852ac12d7fb5b7f4c8bee9a636dd2/lighthouse-core/runner.js#L343-L353)

Reply:
thanks for the explanation @brendankenny. I updated that comment, and removed this error from the proto.